index.bs

```diff
@@ -3265,8 +3265,9 @@ credential.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialUserEntity/displayName}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        [=Authenticators=] MUST accept and store a 64-byte minimum length for a {{PublicKeyCredentialUserEntity/displayName}}
-        member's value. Authenticators MAY truncate a {{PublicKeyCredentialUserEntity/displayName}} member's value so that it fits within 64 bytes. See [[#sctn-strings-truncation]] about truncation and other considerations.
+        [=Authenticators=] MUST accept and store a {{PublicKeyCredentialUserEntity/displayName}} member's value
+        whose UTF-8 encoding is 64 bytes or shorter.
+        Authenticators MAY truncate a {{PublicKeyCredentialUserEntity/displayName}} member's value so that its UTF-8 encoding fits within 64 bytes. See [[#sctn-strings-truncation]] about truncation and other considerations.
```

Comment:
```suggestion
Authenticators MAY truncate characters from the end of a {{PublicKeyCredentialEntity/displayName}} member's value so that when the display name is encoded in UTF-8, its length fits within 64 bytes. See [[#sctn-strings-truncation]] about truncation and other considerations.
```

Reply:
Thanks! I saw some opportunities to make this language both shorter and more strict, please let me know what you think.